components/telemetry-operator/go.mod

```diff
@@ -48,6 +45,7 @@ require (
 	github.com/Azure/go-autorest/tracing v0.6.0 // indirect
 	github.com/PuerkitoBio/purell v1.1.1 // indirect
 	github.com/PuerkitoBio/urlesc v0.0.0-20170810143723-de5bf2ad4578 // indirect
+	github.com/benbjohnson/clock v1.3.0 // indirect
```

Comment:
It looks a bit weird that removing dependencies introduces a new indirect dependency. Maybe run `go mod tidy` once more.

Reply:
Agree, tried it, brings no results :) Probably, we missed it sometime earlier and we didn't have a job that ensures that go.mod is in a tidy state